remove specific date for v0.28 release (#2137)
grafana/agent: README.md

## Release schedule

> Starting with the v0.28 release, a new minor release is planned to be created

Reviewer: ```suggestion
Starting with the v0.28 release, a new minor release is planned for
```
